.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 .idea/
+.bsp/
```

Comment:
Out of interest - what is this directory?

Reply:
It's the cache directory for the Build Server Protocol – it is used by more recent version of the Scala plug-in in IntelliJ ; sbt introduced BSP support in 1.4, I think.